src/api/expense.controller.js

```diff
@@ -0,0 +1,94 @@
+const { expenseService } = require('../services/expense.service.js');
+const { userService } = require('../services/user.service.js');
+
+const getExpenses = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = await expenseService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = async (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || amount <= 0) {
```

Comment:
Consider adding validation for the `category` field to ensure it meets any business logic requirements. Currently, it defaults to an empty string if not provided, but this might not be sufficient if categories are required or need to be validated against a list of allowed values.